src/pspm_sf.m

```diff
@@ -202,8 +206,8 @@
   % 3.5 Get marker data
   if any(strcmp(model.timeunits, {'marker', 'markers'}))
     if options.marker_chan_num
-      [nsts, ~, ndata] = pspm_load_data(model.datafile, options.marker_chan_num);
-      if nsts == -1
+      [sts, ~, ndata] = pspm_load_data(model.datafile, options.marker_chan_num);
```

Comment:
If pspm_sf supports multiple data files then the data file index is missing in line 209.

Reply:
> If pspm_sf supports multiple data files then the data file index is missing in line 209.

I don't quite remember when I wrote this but this is definitely incorrect. Yes I have fixed it now. Thanks for pointing out the problem.